lib/lotus/routing/resource/action.rb

```diff
@@ -221,6 +221,22 @@ def endpoint
         def separator
           @options[:separator]
         end
+
+        # Resource controller name
+        #
+        # @example
+        #   Lotus::Router.new do
+        #     resources 'flowers', controller: 'rocks'
+        #   end
+        #
+        #   # It will mount path 'flowers/new' to Rocks::New instead of Flowers::New
+        #   # Same for other action names
+        #
+        # @api private
+        # @since 0.2.1
```

Comment:
@jodosha Sorry, I did not understood. I see @since there, am I wrote wrong version number?